public/sass/pages/_explore.scss

```diff
@@ -14,6 +22,9 @@
   }
 }
 
+.explore-active-button-glow {
+  box-shadow: $btn-active-box-shadow;
+}
 .explore-ds-picker {
```

Comment:
`icon-brand-gradient` and `explore-active-button-glow` have been created as they are used in different components across explore.